diff --git a/protobuf_deps.bzl b/protobuf_deps.bzl
index b54f12b6..0ae54c70 100644
--- a/protobuf_deps.bzl
+++ b/protobuf_deps.bzl
@@ -1,6 +1,16 @@
 """Load dependencies needed to compile the protobuf library as a 3rd-party consumer."""

 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+
+def com_google_errorprone_error_prone_annotations():
+    jvm_maven_import_external(
+        name = "com_google_errorprone_error_prone_annotations",
+        artifact = "com.google.errorprone:error_prone_annotations:2.3.2",
+        server_urls = ["http://central.maven.org/maven2"],
+        artifact_sha256 = "357cd6cfb067c969226c442451502aee13800a24e950fdfde77bcdb4565a668d",
+        licenses = ["notice"],  # Apache 2.0
+    )

 def protobuf_deps():
     """Loads common dependencies needed to compile the protobuf library."""
@@ -18,3 +28,11 @@ def protobuf_deps():
             strip_prefix = "zlib-1.2.11",
             urls = ["https://zlib.net/zlib-1.2.11.tar.gz"],
         )
+
+    if "com_google_errorprone_error_prone_annotations" not in native.existing_rules():
+        com_google_errorprone_error_prone_annotations()
+
+    native.bind(
+        name = "error_prone_annotations",
+        actual = "@com_google_errorprone_error_prone_annotations//jar",
+    )
