diff --git a/java/image.bzl b/java/image.bzl
index 1203c94..6c16680 100644
--- a/java/image.bzl
+++ b/java/image.bzl
@@ -190,7 +190,7 @@ def _jar_app_layer_impl(ctx):
     args = [ctx.expand_location(arg, ctx.attr.data) for arg in ctx.attr.args]
     jvm_flags = [ctx.expand_location(flag, ctx.attr.data) for flag in ctx.attr.jvm_flags]
 
-    entrypoint = [
+    entrypoint = ctx.attr.entrypoint + [
         "/usr/bin/java",
         "-cp",
         # Support optionally passing the classpath as a file.
@@ -254,6 +254,7 @@ def java_image(
         name,
         base = None,
         main_class = None,
+        entrypoint = [],
         deps = [],
         runtime_deps = [],
         layers = [],
@@ -305,6 +306,7 @@ def java_image(
         name = name,
         base = base,
         binary = binary_name,
+        entrypoint = entrypoint,
         main_class = main_class,
         jvm_flags = jvm_flags,
         deps = deps,