diff --git a/container/BUILD b/container/BUILD
index 8254214..8f2a626 100644
--- a/container/BUILD
+++ b/container/BUILD
@@ -14,6 +14,11 @@
 
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 load("@pip_deps//:requirements.bzl", "requirement")
+load(
+    "@rules_python//python:defs.bzl",
+    "py_binary",
+    "py_library",
+)
 
 package(default_visibility = ["//visibility:public"])
 
diff --git a/toolchains/docker/BUILD b/toolchains/docker/BUILD
index 7c88ceb..89f9025 100644
--- a/toolchains/docker/BUILD
+++ b/toolchains/docker/BUILD
@@ -41,7 +41,7 @@ docker_toolchain(
 toolchain(
     name = "default_linux_toolchain",
     target_compatible_with = [
-        "@bazel_tools//platforms:linux",
+        "@platforms//os:linux",
     ],
     toolchain = "@docker_config//:toolchain",
     toolchain_type = ":toolchain_type",
@@ -50,7 +50,7 @@ toolchain(
 toolchain(
     name = "default_windows_toolchain",
     target_compatible_with = [
-        "@bazel_tools//platforms:windows",
+        "@platforms//os:windows",
     ],
     toolchain = "@docker_config//:toolchain",
     toolchain_type = ":toolchain_type",
@@ -59,7 +59,7 @@ toolchain(
 toolchain(
     name = "default_osx_toolchain",
     target_compatible_with = [
-        "@bazel_tools//platforms:osx",
+        "@platforms//os:osx",
     ],
     toolchain = "@docker_config//:toolchain",
     toolchain_type = ":toolchain_type",