--- repositories/go_repositories.bzl.orig	2023-09-23 08:36:00.148468653 -0400
+++ repositories/go_repositories.bzl	2023-09-23 08:33:22.502127476 -0400
@@ -37,7 +37,7 @@
         go_repository_default_config (str, optional): A file used to determine the root of the workspace.
     """
     go_rules_dependencies()
-    go_register_toolchains()
+    go_register_toolchains("1.21.0")
     gazelle_dependencies(go_repository_default_config = go_repository_default_config)
     excludes = native.existing_rules().keys()
     if "com_github_google_go_containerregistry" not in excludes:
