--- a/google/devtools/build/v1/BUILD.bazel	2026-01-09 14:37:16.000000000 -0500
+++ b/google/devtools/build/v1/BUILD.bazel	2026-01-26 16:08:08.530496707 -0500
@@ -165,16 +165,6 @@
     transport = "grpc",
 )
 
-py_test(
-    name = "build_py_gapic_test",
-    srcs = [
-        "build_py_gapic_pytest.py",
-        "build_py_gapic_test.py",
-    ],
-    legacy_create_init = False,
-    deps = [":build_py_gapic"],
-)
-
 # Open Source Packages
 py_gapic_assembly_pkg(
     name = "devtools-build-v1-py",
