diff --git a/repositories.bzl b/repositories.bzl
index cced1d2..5d87515 100644
--- a/repositories.bzl
+++ b/repositories.bzl
@@ -106,6 +106,10 @@ def grpc_java_repositories():
             name = "com_github_grpc_grpc",
             strip_prefix = "grpc-1.46.0",
             sha256 = "67423a4cd706ce16a88d1549297023f0f9f0d695a96dd684adc21e67b021f9bc",
+            patch_cmds = [
+                "sed -i 's/cfg = \"host\"/cfg = \"exec\"/' generate_cc.bzl",
+                "sed -i 's/cfg = \"host\"/cfg = \"exec\"/' build_deps.bzl",
+            ],
             urls = [
                 "https://github.com/grpc/grpc/archive/v1.46.0.tar.gz",
             ],
